dune-project

```diff
@@ -35,7 +35,7 @@
  (name atd)
  (depends
   (ocaml (>= 4.08))
-  (dune (>= 2.8))
+  dune ; version constraint is added automatically by dune
```

Comment:
you can even remove the mention of `dune` totally. I did it in 028171feb9f5febb9357e32669e2e4fcb4e089ed

Reply:
thx